pkg/preparer/orchestrate.go

```diff
@@ -203,6 +219,11 @@ func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Log
 func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, logger logging.Logger) bool {
 	// do not remove the logger argument, it's not the same as p.Logger
 
+	if p.podIdForbidden(newManifest.ID()) {
```

Comment:
This is interesting. Normally I would put this only in `WatchForPodManifestsForNode`, since it seems wisest to check this as early as possible and not create extra goroutines for the forbidden IDs. I also wanted a test on this, and our tests call `installAndLaunchPod` instead. Perhaps I should figure out some way for tests on `WatchForPodManifestsForNode`?

It would be better to have the check in only one place if it triggers authz failure hooks